dev/src/extension.ts

```diff
@@ -57,6 +57,23 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 
     subscriptions.push(CodewindManager.instance);
 
+    // configure json as the language of the codewind settings file. call setImmediate() to configure any
+    // editor already open at the time the extension is activated.
+    let setSettingsFileLanguage = (doc: vscode.TextDocument) => {
```

Comment:
I would pull this out into a proper function rather than inlining it, it is more readable. otherwise it should be `const`